### PR TITLE
[ADD] [16.0] mail_message_destiny_link_template

### PR DESCRIPTION
**Pull request by Shide**
Added an inverse [message_origin_link](https://github.com/odoo/odoo/blob/16.0/addons/mail/data/mail_templates_chatter.xml#L49-L57) template

Example of how it looks on a sale.order:
image

Usage is detailed in USAGE.md

MT-6090 @moduon @yajo @EmilioPascual @rafaelbn please review if you want :)

### CODE REVIEW

File: mail_message_destiny_link_template/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Mail Message Destiny Link Template",
+    "summary": "Mail template to show destiny records in chatter.",
+    "version": "16.0.1.0.0",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "category": "Tools",
+    "website": "https://github.com/OCA/server-ux",
+    "depends": ["mail"],
```
Comment (yajo) [inline mail_message_destiny_link_template/__manifest__.py:11]: issue: Please add the required maintainers

Comment (yajo) [inline mail_message_destiny_link_template/__manifest__.py:11]: Please add the required maintainers

**Review by EmilioPascual — APPROVED** [review]

**Comment by Shide** [comment]
@yajo Used in https://github.com/OCA/sale-workflow/pull/3214

**Review by yajo — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**